presctl/cmd/add.go

```diff
@@ -0,0 +1,77 @@
+package cmd
+
+import (
+	"net/http"
+
+	"github.com/Microsoft/presidio/presctl/cmd/entities"
+
+	"github.com/spf13/cobra"
+)
+
+// addCmd represents the add command
+var addCmd = &cobra.Command{
+	Use:   "add",
+	Short: "adds a new resource type",
+	Long:  `Use this command to add to Presidio a new resource of the specified type.`,
+}
+
+// templateCmd represents the template command
+var templateCmd = &cobra.Command{
+	Use:   "template [name]",
+	Args:  cobra.MinimumNArgs(1),
+	Short: "adds a new template resource",
+	Long:  `Use this command to add to presidio a new template.`,
```

Comment:
Is this function adding all templates types?

Reply:
yes. depends on the action param (analyze/anonymize) 
@omri374